packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -933,6 +929,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       filteredEntries,
       textDirection,
       focusedIndex: currentHighlight,
+      useMaterial3: Theme.of(context).useMaterial3,
```

Comment:
Nit: Maybe save this up top in a variable and reuse it (for slight performance).